src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -80,6 +80,7 @@ static std::string          subscribeContextCollectionName;
 static std::string          subscribeContextAvailabilityCollectionName;
 static std::string          assocationsCollectionName;
 static Notifier*            notifier;
+static bool                 multitenant;
```

Comment:
Because of encapsulation. I'd prefer all the variables used in mongoBackend to be keep in mongoBackend. Thus, we use `multitenant` in mongoBackend, initialized with the value of `mtenant`.